packages/dds/sequence/src/revertibles.ts

```diff
@@ -453,8 +453,9 @@ function revertLocalChange(
 	const startSlidePos = getSlidePosition(string, revertible.start, start);
 	const end = string.localReferencePositionToPosition(revertible.end);
 	const endSlidePos = getSlidePosition(string, revertible.end, end);
-	if (!isValidRange(startSlidePos, endSlidePos, string)) return;
-	collection.change(id, startSlidePos, endSlidePos);
+	if (isValidRange(startSlidePos, endSlidePos, string)) {
+		collection.change(id, startSlidePos, endSlidePos);
+	}
```

Comment:
This change is not related to the rest of the PR. I happened to notice that revertible local refs aren't being cleaned up in this case.

Reply:
This makes more sense than what I had before anyway. I probably had something similar in revertLocalDelete, so if you want to change that instance as well, go ahead. If not, I can make the change as part of [AB#4859](https://dev.azure.com/fluidframework/internal/_workitems/edit/4859).